Source: Arch Linux
Upstream: No, distro specific
Reason: avoids conflicts with coreutils binaries like md5sum

--- Makefile.am	2018-01-12 22:41:55.000000000 +0100
+++ Makefile.am	2018-01-25 15:34:35.649902674 +0100
@@ -131,8 +131,8 @@
 
 install-exec-hook:
 	(cd $(DESTDIR)$(bindir) && \
-	($(foreach link, $(kcapi_hasher_links), ln -f kcapi-hasher $(link);)) && \
-	($(foreach link, $(kcapi_hasher_links), openssl sha256 -hmac orboDeJITITejsirpADONivirpUkvarP $(link) > .$(link).hmac;)))
+	($(foreach link, $(kcapi_hasher_links), ln -f kcapi-hasher $(link)-kcapi;)) && \
+	($(foreach link, $(kcapi_hasher_links), openssl sha256 -hmac orboDeJITITejsirpADONivirpUkvarP $(link)-kcapi > .$(link)-kcapi.hmac;)))
 endif
 
 if ENABLE_KCAPI_RNGAPP

